react/hooks/useInterval.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react'
+
+/**
+ * Force a new render each `duration` ms
+ *
+ * @param {number} duration - in millisecond
+ * @returns {Date} last trigger date for a rerender
+ */
+export default function useInterval(duration) {
```

Comment:
Have you considered a name like `setIntervalRerender` or something similar? To make it explicit that the hook is about re-rendering and not so much about intervals.

Reply:
usePeriodicRender